.eslintrc.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+	root: true,
+	env: {
+		browser: true,
+		es6: true,
+		node: true,
+		jest: true
+	},
+	globals: {
+		t: true,
+		n: true,
+		OC: true,
+		OCA: true,
+		Vue: true,
+		VueRouter: true
+	},
+	parserOptions: {
+		parser: 'babel-eslint'
+	},
+	extends: [
+		'eslint:recommended',
+		'plugin:node/recommended',
+		'plugin:vue/essential',
+		'plugin:vue/recommended',
+		'standard'
```

Comment:
I'm not so fond of `standard` (but no worries). Maybe it it should be - or not - inside Nextcloud coding style ? Do you want me to open an issue on server ?

Reply:
I just copy pasted the basic stuff from the contacts app, happy to change it

Reply:
@tcitworld The overall edits I made on the standard really feels like what we usually encounter on the nextcloud apps :)
Could you take a look at the contacts app?

Reply:
@tcitworld btw, if you decide for a better settings that fit the way we usually use on our apps (we globally have a similar coding style between our apps :wink:) could you ping me so I can take a look and also update the contacts app to the same settings?